dask_mongo/core.py

```diff
@@ -44,41 +43,27 @@ def to_mongo(
     collection:
       Name of the collection within the database to write to.
       If it does not exists it will be created.
-    compute_options:
-      Keyword arguments to be forwarded to ``dask.compute()``.
+    Returns
+    -------
+    dask.delayed
     """
-    if compute_options is None:
-        compute_options = {}
-
-    partitions = [
-        write_mongo(partition, connection_args, database, collection)
-        for partition in bag.to_delayed()
-    ]
+    partials = bag.map_partitions(write_mongo, connection_args, database, collection)
+    return checkpoint(partials)
```

Comment:
Can I ask you to add a `compute=` keyword argument (which is `True` by default) and call `.compute()` accordingly in this function? That way `to_mongo` will be consistent with other `to_parquet`, `to_csv`, `to_avro` methods in Dask.

Reply:
done